mantle/cmd/ore/kubevirt/build.go

```diff
@@ -0,0 +1,109 @@
+package kubevirt
+
+import (
+	"archive/tar"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/empty"
+	"github.com/google/go-containerregistry/pkg/v1/mutate"
+	"github.com/google/go-containerregistry/pkg/v1/tarball"
+)
+
+// BuildContainerDisk creates an image complying with a KubeVirt containerDisk format
+// (https://kubevirt.io/user-guide/virtual_machines/disks_and_volumes/#containerdisk-workflow-example).
+// It creates an image based on this Dockerfile receipt:
+//   > FROM scratch
+//   > ADD --chown=107:107 mydisk.qcow2 /disk/
```

Comment:
Have you toyed at all with using buildah instead for this as suggested earlier? Less vendoring is a win. :)

But also fine with this too.

Reply:
I see where this is going. I can definitely understand that desire. I will try it again :)

Reply:
Done.